integration/testrunner/logging.go

```diff
@@ -27,6 +27,11 @@ func MakeAuditLogRequest(id, endpointURL string, requestTimeout time.Duration, a
 		return nil, fmt.Errorf("failed to create audit log http request: %w", err)
 	}
 
+	signedToken, err := justificationToken("talker-app")
```

Comment:
This is used in http requests only, should be logging shell app instead? also wondering why we need this as audience, when verify we only check if justification is non-nil?

Reply:
This is a typo. I changed it to `logging shell`. We didn't need this when validating justification. But previously grpc set the audience to `talker-app`. I don't want to change the existing behavior.

Reply:
Ok, another way is to use a dummy or no audience at all so that we don't need to pass any thing that we don't need.